easy-coding-standard.neon

```diff
@@ -0,0 +1,10 @@
+# this will be placed in doctrine/coding-stanard, easy-accessible for anyone
```

Comment:
Syntax is mostly the same with YAML.
NEON simply [allows class autocomplete](https://github.com/Symplify/EasyCodingStandard#dont-write-checker-classes-make-use-of-neon-plugin).